format.go

```diff
@@ -114,3 +120,27 @@ func isHostName(s string) bool {
 
 	return ok
 }
+
+func isPasswordAa0(pw string) bool {
+	large := false
+	small := false
+	number := false
+
+	for _, r := range pw {
+		switch {
+		case 'a' <= r && r <= 'z':
+			small = true
+		case 'A' <= r && r <= 'Z':
+			large = true
+		case '0' <= r && r <= '9':
+			number = true
+		case ('!' <= r && r <= '/') || (':' <= r && r <= '@') || ('[' <= r && r <= '`') || ('{' <= r && r <= '~'):
```

Comment:
1. Make this conditional statement simpler.
2. If ignoring any condition, give some comment for explanation.

```go
case ("!" <= r && r <= "~"):
    // Do nothing, because ASCII characters are allowed.
```